share/python/driver.py

```diff
@@ -1138,6 +1805,11 @@ def optimize(name, **kwargs):
 
     # Make sure the molecule the user provided is the active one
     molecule = kwargs.pop('molecule', psi4.get_active_molecule())
+
+    # If we are feezing cartesian, do not orient or COM
+    if psi4.get_local_option("OPTKING", "FROZEN_CARTESIAN"):
+        molecule.fix_orientation(True)
+        molecule.fix_com(True)   
```

Comment:
"Frozen Cartesian" isn't strictly a boolean, so just checking that you checked that the "if" statement does what you expect.